src/components/networked.js

```diff
@@ -373,39 +381,42 @@ AFRAME.registerComponent('networked', {
   },
 
   updateComponent: function (el, componentName, data) {
-    if(!NAF.options.useLerp) {
+    const transformComponents = ["position", "rotation", "scale"]
+    if(!NAF.options.useLerp || !transformComponents.includes(componentName)) {
       el.setAttribute(componentName, data);
       return;
     }
 
-    var buffer = null;
-    var interpolationBuffer = this.interpolationBuffers.find((item) => item.el === el);
-    if (!interpolationBuffer) {
-      buffer = new InterpolationBuffer(InterpolationBuffer.MODE_LERP, 0.1);
-      this.interpolationBuffers.push({ buffer: buffer, el: el });
-    } else {
-      buffer = this.interpolationBuffers.find((item) => item.el === el).buffer;
+    var bufferInfo = this.bufferInfos.find((info) => info.object3D === el.object3D);
+    if (!bufferInfo) {
+      bufferInfo = { buffer: new InterpolationBuffer(InterpolationBuffer.MODE_LERP, 0.1),
+                     object3D: el.object3D,
+                     componentNames: [componentName] };
+      this.bufferInfos.push(bufferInfo);
+    }
+    var buffer = bufferInfo.buffer;
+    var componentNames = bufferInfo.componentNames;
+    if (!componentNames.includes(componentName)){
+      componentNames.push(componentName);
```

Comment:
This code could be slightly more precise if you put this check into an `else` clause, to run only in the case where there was already an interpolation buffer, but it's such a small difference that it's hard for me to care.